content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md

```diff
@@ -0,0 +1,117 @@
+
+![GooglePhotosAPICreateExternal](/images/SCALE/22.12/GooglePhotosAPICreateExternal.png "Create External")
+
+Enter an **App name** and **User support email**. 
+
+![GooglePhotosAPIAppInformation](/images/SCALE/22.12/GooglePhotosAPIAppInformation.png "Enter App Information")
+
+Enter an email address in the **Developer contact information** section, then click **SAVE AND CONTINUE**.
+
+![GooglePhotosAPIDeveloperContactInformation](/images/SCALE/22.12/GooglePhotosAPIDeveloperContactInformation.png "Enter Developer Contact Information")
+
+Continue to the **Add users** section, enter your email address, then click **ADD**.
+
+![GooglePhotosAPIAddTestingUser](/images/SCALE/22.12/GooglePhotosAPIAddTestingUser.png "Add Testing User")
+
+On the **OAuth consent screen**, click **PUBLISH APP** under **Testing** and push the app to production.
+
+![GooglePhotosAPIPublish](/images/SCALE/22.12/GooglePhotosAPIPublish.png "Publish Status")
+
+{{< expand "Can I leave the app in testing mode?" "v" >}}
+You can leave the app in testing mode, but your cloud sync task will fail after your testing app credentials expire after seven days. 
+{{< /expand >}}
+
+### Create Credentials
+
+Click **Credentials** on the left menu, then click **+ CREATE CREDENTIALS** and select **OAuth client ID**.
+
+![GooglePhotosAPIAddCredentials](/images/SCALE/22.12/GooglePhotosAPIAddCredentials.png "Add Credentials")
+
+Select **Desktop app** in the **Application type** dropdown, then enter a name for the client ID and click **CREATE**.
+
+![GooglePhotosAPICreateOAuthClientID](/images/SCALE/22.12/GooglePhotosAPICreateOAuthClientID.png "Create OAuth Client ID")
+
+Copy and save your client ID and secret, or download the JSON file.
+
+![GooglePhotosAPICopyIDandSecret](/images/SCALE/22.12/GooglePhotosAPICopyIDandSecret.png "Copy ID and Secret")
+
+## Configuring Rclone
+
+Download [rclone](https://rclone.org/downloads/) for your OS and open it in a command line utility. The example photos in this article use Powershell in Windows OS.
+
+Enter `rclone config`, then enter `n` to create a new remote.
+
+![GooglePhotosAPIrcloneConfig](/images/SCALE/22.12/GooglePhotosAPIrcloneConfig.png "Configure rclone")
+
+Enter a name for the new remote, then enter the number from the list corresponding to Google Photos.
+
+Enter the client id and secret you saved when you created the Google Photos API credentials, then enter `false` to keep the Google Photos backend read-only.
+
+![GooglePhotosAPIrcloneConfig2](/images/SCALE/22.12/GooglePhotosAPIrcloneConfig2.png "Configure rclone")
+
+Do not edit the advanced config. When prompted about automatically authenticating rclone with the remote, enter `y`.
+
+A browser window opens to authorize rclone access. Click **Allow*.
+
+In the command line, enter `y` when prompted about media item resolution to complete the configuration.
+
+Copy and save the Options (type, client_id, client_secret, token), then enter `y` to keep the new remote.
```

Comment:
is Options something on the screen? if yes use bold, if not or a general reference to options use lower case